src/pages/index.js

```diff
@@ -50,7 +50,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; A JavaScript library for building user interfaces"
+          title="ری ایکٹ &ndash; یے جاوا اسکرپٹ کی لا ئبریری ھے جو یوزر انٹرفیسس بنانے میں کا م آ تی ھے"
```

Comment:
Do not translate the words that represent products i.e. React, JavaScript etc. Read the [universal style guide](https://github.com/reactjs/reactjs.org-translation/blob/master/style-guide.md) and see examples from other translation projects such as [React docs in Arabic](https://ar.reactjs.org).